presto-main/src/main/java/com/facebook/presto/operator/scalar/RegexpFunctions.java

```diff
@@ -195,16 +201,18 @@ private static void appendReplacement(SliceOutput result, Slice source, Regex pa
 
     @Description("string(s) extracted using the given pattern")
     @ScalarFunction
+    @LiteralParameters("x")
     @SqlType("array(varchar)")
```

Comment:
Actually i think it is even possible to put there `varchar(x)`, by analogy to `split` function